unattended_scripts/wazuh_install.sh

```diff
@@ -261,8 +344,11 @@ main() {
         importFunction "wazuh.sh"
         importFunction "filebeat.sh"
 
+        progressbar_total=8
```

Comment:
Same

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/1080/commits/b6529a2b3b0eeb287bd573b77284f34b2fef99a6